.vscode/settings.json

```diff
@@ -1,5 +1,7 @@
 // Place your settings in this file to overwrite default and user settings.
 {
+    "editor.tabSize": 4,
+    "editor.insertSpaces": false,
```

Comment:
I actually just noticed there was a weird mismatch in my files.. I think I set `insertSpaces: false` but the default behaviour scans the existing file and my `extension.ts` (and somehow `util.ts`) have spaces.

4-char tabs is my preference and I planned on sorting this

:+1: !